### PR TITLE
feat(FEC-8754): PiP Accessibility

### PR DESCRIPTION
**Pull request by shalommeoded**
### Description of the Changes

* Created a shortcut to the PiP plugin and assigned to the `P` key.
* Added the Keyboard event to the event-dispatcher.

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @shalommeoded,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: src/components/keyboard/keyboard.js [hunk 1]
```diff
@@ -136,6 +136,13 @@ class KeyboardControl extends BaseComponent {
         return true;
       }
     },
+    [KeyMap.P]: () => {
+      if (!this.player.isInPictureInPicture()) {
```
Comment (OrenMe) [inline src/components/keyboard/keyboard.js]: if you are in focus on screen can we toggle back to out of PiP? if so add the `else` clause for it

Comment (shalommeoded) [reply src/components/keyboard/keyboard.js]: @OrenMe I can see that that the option to exit PiP is available and working via script (console), however, the limitation is on the keybinding, it seems that when on PiP mode, the player bindings are getting overridden.

Following the above, do you still want me to implement the `else` clause?

Comment (OrenMe) [reply src/components/keyboard/keyboard.js]: ok, but if you focus back on page and click it will toggle? if so please add it

Comment (shalommeoded) [reply src/components/keyboard/keyboard.js]: Added

**Pull request merged**